packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx

```diff
@@ -45,7 +47,14 @@ class ClearStoreErrorBoundary extends Component<
             <ModalButtonContainer data-testid="modal-button-container">
               <DarkFunctionButton
                 onClick={async () => {
-                  await persistor.purge();
+                  if (this.props.router?.query?.app === "Downloads") {
```

Comment:
The solution is currently adequate since only a few apps have their own data storage. However, as the number of apps increases, we may need to revisit this.